rust/signed_doc/src/builder/cbor_map.rs

```diff
@@ -0,0 +1,49 @@
+use std::collections::BTreeMap;
+
+use super::EncodeError;
+
+/// A map of CBOR encoded key-value pairs with **bytewise** lexicographic key ordering.
+#[derive(Debug, Default)]
+pub struct CborMap(BTreeMap<Vec<u8>, Vec<u8>>);
```

Comment:
It seems redundant to have a separate type for it, I think it would be enough to have a type alias for it